mitmproxy/tools/web/app.py

```diff
@@ -266,6 +272,12 @@ def post(self):
         bio.close()
 
 
+class ExportFlow(RequestHandler):
```

Comment:
I think right after this PR is complete we should add an API endpoint to execute commands. We can then get rid of this one as it's basically the same as invoking a particular command. But that's work for the next PR, not this one. 😃